modules/exporter/citation_exporter.module

```diff
@@ -15,12 +15,22 @@ require_once dirname(__FILE__) . '/includes/batch.inc';
  */
 define('CITATION_EXPORTER_INDEX_SESSION', 'citation_exporter');
 define('CITATION_EXPORTER_EXPORT_MENU', 'islandora/citation/export');
+define('CITATION_EXPORTER_ADMIN_MENU', 'admin/islandora/tools/citation/export/config');
```

Comment:
I don't think this path needs to be so deep. Why not something more like `admin/islandora/tools/citaiton_export`